src/kubeobjects/daemonset_test.go

```diff
@@ -12,95 +12,74 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const (
```

Comment:
the diff looks scary, but I just refactored this according to the guidelines

Reply:
nicely refactored